mappings/net/minecraft/client/gui/hud/debug/DebugHud.mapping

```diff
@@ -1,49 +1,85 @@
-CLASS net/minecraft/unmapped/C_epxplegp net/minecraft/client/gui/hud/DebugHud
+CLASS net/minecraft/unmapped/C_epxplegp net/minecraft/client/gui/hud/debug/DebugHud
+	FIELD f_ccxylgcn frameRateSampleLog Lnet/minecraft/unmapped/C_fnzofwlw;
+	FIELD f_cdiumrmi pingSampleLog Lnet/minecraft/unmapped/C_fnzofwlw;
 	FIELD f_demcwxlr client Lnet/minecraft/unmapped/C_ayfeobid;
 	FIELD f_fjhfdzhz HEIGHT_MAP_TYPES Ljava/util/Map;
 	FIELD f_hdjainem fluidHit Lnet/minecraft/unmapped/C_eetoegzn;
+	FIELD f_jwptaljh showingCharts Z
+	FIELD f_kggcezmz frameRateChartRenderer Lnet/minecraft/unmapped/C_vgifhjob;
+	FIELD f_llewhquv tickRateSampleLog Lnet/minecraft/unmapped/C_fnzofwlw;
+	FIELD f_lmmlpwdr pingChartRenderer Lnet/minecraft/unmapped/C_ftwidtyt;
+	FIELD f_mriubqfx tickRateChartRenderer Lnet/minecraft/unmapped/C_rggqladj;
+	FIELD f_nggbpvpt bandwidthUsageChartRenderer Lnet/minecraft/unmapped/C_zjhfcbxz;
 	FIELD f_nivhpjpa TEXT_COLOR I
 	FIELD f_qympybag allocationRateCalculator Lnet/minecraft/unmapped/C_epxplegp$C_itbnpmom;
+	FIELD f_rhtdusnd bandwidthUsageSampleLog Lnet/minecraft/unmapped/C_fnzofwlw;
+	FIELD f_rlumovzv showProfilerChart Z
 	FIELD f_rmhsdqts MARGIN_LEFT I
 	FIELD f_tekegaef MARGIN_RIGHT I
 	FIELD f_unrhvpxp MARGIN_TOP I
 	FIELD f_uzvvfrgc pos Lnet/minecraft/unmapped/C_ynrszrtu;
 	FIELD f_vbjupvgy blockHit Lnet/minecraft/unmapped/C_eetoegzn;
 	FIELD f_vmhafjuy chunk Lnet/minecraft/unmapped/C_hrdsvlkq;
 	FIELD f_warxjbak chunkFuture Ljava/util/concurrent/CompletableFuture;
+	FIELD f_xemytmmw showingFpsTpsCharts Z
```

Comment:
`show`/`showing` is inconsistent, I prefer `show`.
Also not a fan of `FpsTps`
```suggestion
	FIELD f_xemytmmw showFpsAndTpsCharts Z
```
or
```suggestion
	FIELD f_xemytmmw showPerSecCharts Z
```
or
```suggestion
	FIELD f_xemytmmw showPerSecondsCharts Z
```
or
```suggestion
	FIELD f_xemytmmw showRateCharts Z
```

Reply:
I think rate charts might work... I don't like any of the options though really (including fps TPS)